build.gradle

```diff
@@ -26,21 +26,24 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
 	implementation 'org.springframework.boot:spring-boot-starter-validation'
-	implementation 'org.springframework.boot:spring-boot-starter-security'
-	implementation 'org.springframework.security:spring-security-oauth2-client:5.7.5'
-	implementation 'org.springframework.security:spring-security-oauth2-jose:5.7.5'
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
-	implementation 'mysql:mysql-connector-java:8.0.31'
-	implementation 'org.flywaydb:flyway-core:9.8.3'
-	implementation 'org.flywaydb:flyway-mysql:9.8.3'
+	implementation 'org.springframework.boot:spring-boot-starter-security:3.0.1'
+	implementation 'org.springdoc:springdoc-openapi-ui:1.6.14'
+	implementation 'org.flywaydb:flyway-mysql:9.11.0'
+	implementation 'org.flywaydb:flyway-core:9.11.0'
+	implementation 'io.jsonwebtoken:jjwt-api:0.11.5'
+	implementation 'io.jsonwebtoken:jjwt-impl:0.11.5'
+	implementation 'io.jsonwebtoken:jjwt-jackson:0.11.5'
+	compileOnly 'org.projectlombok:lombok'
+	runtimeOnly 'com.mysql:mysql-connector-j'
```

Comment:
why do you need it? is spring boot starter not enough?

Reply:
Yes, I need it. Because without this dependency Tomcat context doesn`t started.